src/components/Messages/Message/index.vue

```diff
@@ -126,7 +126,7 @@
   </li>
 </template>
 <script>
-import { mapActions } from 'vuex'
+import { mapGetters, mapActions } from 'vuex'
```

Comment:
No. I told you that these components should be as simple as possible. The same goes for below's `created ()` hook.